test_scripts/Defects/7_1/3620/common.lua

```diff
@@ -0,0 +1,102 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require("test_scripts/Protocol/commonProtocol")
+local constants = require("protocol_handler/ford_protocol_constants")
+
+--[[ General configuration parameters ]]
+common.testSettings.isSelfIncluded = false
+config.defaultProtocolVersion = 5
+
+--[[ Common Variables ]]
+common.SERVICE_TYPE = constants.SERVICE_TYPE
+common.FRAME_TYPE = constants.FRAME_TYPE
+common.FRAME_INFO = constants.FRAME_INFO
+
+--[[ Common Functions ]]
+function common.startServiceNACKwithNonExistedSessionId(pServiceParams, isEncrypted)
+  isEncrypted = isEncrypted or false
+  local protectionLable = isEncrypted and "a protected" or "an unprotected"
+  local nonExistedSessionId = 5
+  local mobSession = common.getMobileSession()
+  mobSession.sessionId = nonExistedSessionId
+  local msg = {
+    serviceType = pServiceParams.serviceType,
+    frameType = common.FRAME_TYPE.CONTROL_FRAME,
+    frameInfo = common.FRAME_INFO.START_SERVICE,
+    encryption = isEncrypted,
+    binaryData = common.bson_to_bytes(pServiceParams.reqParams)
+  }
+  mobSession:Send(msg)
+  mobSession:ExpectControlMessage(pServiceParams.serviceType, {
+    frameInfo = common.FRAME_INFO.START_SERVICE_NACK,
+    encryption = false
+  })
+  :ValidIf(function(_, data)
+      local exp = {
+        reason = {
+          type = common.bsonType.STRING,
+          value = "Cannot start " .. protectionLable .. " service of type " .. pServiceParams.serviceType
+            .. ". Session " .. nonExistedSessionId .. " not found for connection 1"
+        }
+      }
```

Comment:
I would move expected result to the scripts `2` and `5` (as it's currently done for scripts `1`, `3`, `4`)